c8/fabric.py

```diff
@@ -633,6 +633,56 @@ def response_handler(resp):
 
         return self._execute(request, response_handler)
 
+    def collection_figures(self, collection_name):
+        """Returns an object containing statistics about a collection.
+
+        :param collection_name: Collection name.
+        :type collection_name: str | unicode
+        """
+
+        request = Request(
+            method='get',
+            endpoint=f'/collection/{collection_name}/figures',
```

Comment:
This is a good way to work with strings; I also prefer it. But f strings are supported by Python 3.6+. As you can see from our docs, we currently support 3.4+ versions and our code base is filled with `.format` notation such as `string_in_string = "Shepherd {} is on duty.".format("Mary")`. We should use that notation for the time being.